### PR TITLE
Feature/focus reflow fix

### PR DESCRIPTION
**Pull request by bh412**

### CODE REVIEW

File: src/scss/focus.scss [hunk 1]
```diff
@@ -55,15 +55,15 @@
...
 
 .btn-secondary:focus {
-  box-shadow: none;
-  border: 0.2rem solid black !important;
+  box-shadow: 0 0 0 2pt black !important;
   border-radius: 8px;
+  outline: none !important;
 }
 
 .btn-danger:focus {
```
Comment (mlt47) [inline src/scss/focus.scss]: Let's make the change here too while we're at it

File: src/scss/focus.scss [hunk 2]
```diff
@@ -78,25 +78,17 @@
...
-  border: 0.2rem solid $secondary;
-  border-radius: 8px;
-}
-
-.btn.btn-sm.a-alt:focus {
-  box-shadow: none;
-  border: 0.2rem solid $focus-blue;
-  border-radius: 5px;
+  border-color: $secondary !important;
 }
```
Comment (mlt47) [inline src/scss/focus.scss]: Why has this one been deleted?

File: src/scss/focus.scss [hunk 3]
```diff
@@ -78,25 +78,17 @@
...
+  border-radius: 3px;
+  outline: none !important;
 }
 
 .footerTop a:focus {
-  border: 0.2rem solid white;
+  outline: 0.2rem solid white;
 }
 
 // Close modal button
```
Comment (mlt47) [inline src/scss/focus.scss]: I imagine this close button's focus is fine but it is worth a check if it hasn't already been checked

Comment (bh412) [reply src/scss/focus.scss]: all good

File: src/scss/focus.scss (hunk 2 above)
Comment (bh412) [reply src/scss/focus.scss]: It didn't appear to affect anything, but I've added it back incase

**Pull request merged**